configuration/config-txt/overclocking.md

```diff
@@ -44,14 +75,20 @@ This table describes the overvoltage settings for the various Pi models. The fir
 | Pi 3 | 1.2-1.3125V | 0 |
 | Pi Zero | 1.35V | 6 |
 
-**Specific to Pi 4B**: The `core_freq` of the Raspberry Pi 4 can change from the default if either `hdmi_enable_4kp60` or `enable_tvout` are used, due to relationship between internal clocks and the particular requirements of the requested display modes.
+#### Specific to Pi 4B
+
+The `core_freq` of the Raspberry Pi 4 can change from the default if either `hdmi_enable_4kp60` or `enable_tvout` are used, due to relationship between internal clocks and the particular requirements of the requested display modes.
 
 |Display option | Frequency |
 |---------------|-----------|
 | Default        | 500 |     
-| enable_tvout | 432 |
+| enable_tvout | 360 |
 | hdmi_enable_4kp60 | 550 |
 
+Changing `core_freq` in `config.txt` is not supported on the Pi 4, any change from the default will almost certainly cause a failure to boot.
+
+It is recommended when overclocking to use the individual frequency settings rather than `gpu_freq`, as since it sets `core_freq` it is not likely to have the desired effect. 
```

Comment:
Might be worth clarifying this slightly as "... to use the individual frequency settings (`h264_freq`, `isp_freq`, etc.) rather than `gpu_freq`, ..." ?